src/utils/forminputhandling.ts

```diff
@@ -2,8 +2,10 @@ import { RowType } from "@/types/types";
 import { instance } from "./RootClient";
 
 // get data from the db
-async function getData(page = 1, limit = 5) {
-  const { data } = await instance.get(`/quiz?_page=${page}&_limit=${limit}`);
+async function getData(page: number, limit: number) {
+  const { data } = await instance.get(
+    `api/session?session_id_is_null=true&offset=${page}&limit=${limit}`
```

Comment:
you will need two functions. one where session id is null, and one where you make request directly to `api/session?offset=${page}&limit=${limit}`

and then process results. 

also u need to display only quizzes for quiz creator part. that is, where platform variable is set to "quiz". for now, display those where platform variable is set to "meet" too. but eventually, we need a filter for this. 

when we expand quiz creator to general session creator, we will have another tab wherein we display only "meet" platform sessions